modules/gapi/CMakeLists.txt

```diff
@@ -111,6 +112,7 @@ set(gapi_srcs
     src/backends/cpu/gcpubackend.cpp
     src/backends/cpu/gcpukernel.cpp
     src/backends/cpu/gcpuimgproc.cpp
+    src/backends/cpu/gcpustereo.cpp
```

Comment:
I believe it is not the only change....

Here we add a dependency on `calib3d` OpenCV module which we need to specify clearly.
Also, this dependency should be optional so if the user explicitly disables that module in OpenCV, G-API should still build (but with Stereo functionality disabled).

Please check it with @AsyaPronina / @TolyaTalamanov

Reply:
Fixed 

Checked on my Linux machine, should works good